tests/load/test_load_report.py

```diff
@@ -1,114 +0,0 @@
-import pytest
-
-from scout.exceptions import DataNotFoundError, IntegrityError
-from scout.load.report import load_cnv_report, load_coverage_qc_report, load_delivery_report
-
-
-def test_load_delivery_report_bad_case_id(adapter):
-
-    ## GIVEN no cases in database
-    assert adapter.case_collection.find_one() is None
-
-    ## WHEN trying to load a report for a case_id that does not exist in the data base
-    case_id = "id_of_non_existing_case"
-    report_path = "a_fakey_path"
-
-    ## THEN an exception should be raised
-    with pytest.raises(DataNotFoundError):
-        load_delivery_report(adapter=adapter, case_id=case_id, report_path=report_path)
-
-
-def test_load_delivery_report_using_case_id_without_update_fail(adapter, case_obj):
```

Comment:
Couldn't we refactor at least one test with an expected fail?

Reply:
I've removed the --update option so the command scout load report <whichever-report> will always update case and never fail. I was reasoning that if you use the command you probably want to update the report anyway. And if you update it wrongly you can always re-update it later